app/views/tracks/build/show.html.haml

```diff
@@ -295,16 +295,18 @@
         - @status.volunteers.users[0, 3].to_a.each do |volunteer|
           .contributor-credit-block
             .contributor
-              = avatar(volunteer)
+              = avatar(volunteer, css_class: "h-32 w-32")
```

Comment:
```suggestion
              = avatar(volunteer, css_class: "h-[32px] w-[32px]")
```

Also below. Or edit the tailwind file to have these?

Reply:
I added this to the Tailwind config.
I still had the Integral-site tailwind setup in my cell-memory where height wasn't overridden. :D